front/src/modules/activities/notes/components/NoteCard.tsx

```diff
@@ -71,17 +76,24 @@ export const NoteCard = ({
     ?.content.map((x: any) => x.text)
     .join('\n');
 
+  const fieldContext = useMemo(
+    () => ({ recoilScopeId: note?.id ?? '' }),
+    [note?.id],
+  );
+
```

Comment:
Is useMemo really needed here? @charlesBochet

Reply:
The prop's value is an object, and every time the parent re-renders, it appears as if the value has changed even if it hasn't. Therefore, we use `useMemo`. As for its necessity, it's debatable.